packages/store/src/patterns/interface-tools.js

```diff
@@ -4,10 +4,23 @@ import { listDifference, objectMap } from '@agoric/internal';
 
 import { fit } from './patternMatchers.js';
 
+/* Destructuring the object assert into two variables, details and quote. */
 const { details: X, quote: q } = assert;
+/* Using the Reflect API to apply a function to an object. */
 const { apply, ownKeys } = Reflect;
+/* Using object destructuring to assign the properties of the object Object to the variables
+defineProperties, seal, and freeze. */
 const { defineProperties, seal, freeze } = Object;
```

Comment:
This kind of commentary is appropriate in a tutorial to walk through code and explain each line. It is not appropriate for inclusion in the production code itself. Once oriented in reading the code itself, the comments simply restate what the code clearly says without any additional insight, and so only add noise from the pov of the production programmer.

That said, everything these comments say is correct, so you're not misunderstanding things.